skypy/power_spectrum/growth.py

```diff
@@ -48,8 +48,8 @@ def growth_function_carroll(redshift, cosmology):
     >>> from astropy.cosmology import default_cosmology
     >>> redshift = np.array([0, 1, 2])
     >>> cosmology = default_cosmology.get()
-    >>> growth_function_carroll(redshift, cosmology)
-    array([0.78136173, 0.47628062, 0.32754955])
+    >>> growth_function_carroll(redshift, cosmology) # doctest: +ELLIPSIS
```

Comment:
I think `# doctest: +ELLIPSIS` is unnecessary because the `ELLIPSIS` option is enable by default though [`pytest-doctestplus`](https://github.com/astropy/pytest-doctestplus).